feature-libs/pickup-in-store/root/components/container/pickup-in-store-order-consignment/pickup-in-store-order-consignment-container.component.ts

```diff
@@ -4,33 +4,42 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { Component, OnInit, Optional } from '@angular/core';
+import { Component, OnDestroy, OnInit, Optional } from '@angular/core';
 import { Consignment, Order } from '@spartacus/order/root';
 import { OutletContextData } from '@spartacus/storefront';
+import { Subscription } from 'rxjs';
 import { tap } from 'rxjs/operators';
+
+export type IOutletContextData = { item: Consignment; order: Order };
```

Comment:
Can you check  `IOutletContextData` in develop branch? I remembered `order` was removed.

Reply:
yes right, these changed are part of https://github.com/SAP/spartacus/pull/17176 and will be added once it's merged to the release branch as well. They are merged as part of the different changes to develop hence we are replicating the same as asked.